articles/network-watcher/traffic-analytics-faq.md

```diff
@@ -191,7 +191,10 @@ armclient post "https://management.azure.com/subscriptions/<NSG subscription id>
 
 ## How is traffic analytics priced?
 
-Traffic analytics is metered. The metering is based on processing of flow log data by the service, and storing the resulting enhanced logs in a Log Analytics workspace. For more details, see the [pricing plan](https://azure.microsoft.com/en-us/pricing/details/network-watcher/). 
+Traffic analytics is metered. The metering is based on processing of flow log data by the service, and storing the resulting enhanced logs in a Log Analytics workspace. 
+
```

Comment:
I recommend putting this as a FAQ in the pricing article ([example](https://azure.microsoft.com/details/expressroute/)), rather than in the technical FAQ. I don't control the pricing article, so you'll need to work with marketing on that change, if you follow my suggestion. You could just add a link above to the pricing article and refer people there for details.